src/file-1.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44133/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44133/project/issues?id=sample&pullRequest=7013&issues=AWmV0YUcf5mRuAnor-4b&open=AWmV0YUcf5mRuAnor-4b)

[ISSUE_KEY]: AWmV0YUcf5mRuAnor-4b